cmd/kyverno/main.go

```diff
@@ -282,6 +286,7 @@ func main() {
 	flagset.DurationVar(&leaderElectionRetryPeriod, "leaderElectionRetryPeriod", leaderelection.DefaultRetryPeriod, "Configure leader election retry period.")
 	flagset.StringVar(&exceptionNamespace, "exceptionNamespace", "", "Configure the namespace to accept PolicyExceptions.")
 	flagset.BoolVar(&enablePolicyException, "enablePolicyException", false, "Enable PolicyException feature.")
+	flagset.IntVar(&servicePort, "servicePort", 443, "Kyverno service port.")
```

Comment:
If we are adding a command line flag, consider expanding the help message to clearly disambiguate this from the port on which the Kyverno webhook actually listens (9443).

Without prior knowledge of this, it would have been hard for me to guess that this referred to the port of the Kyverno Service object and not the port where Kyverno container listens.